docs/layouts/page/playground.html

```diff
@@ -1,12 +1,33 @@
 {{ define "header" }}
 {{ partial "header.html" . }}
+
 {{ end }}
 
+
 {{ define "main" }}
+<style>
+    /* move scroll into playground area */
+    html {
+        overflow-y: hidden;
+    }
+
+    /* body is overflow hidden, so it needs to fill the entire screen height */
+    body {
+        height: 100vh;
+    }
+
+    /* #root comes from storybook, and the first div child has 100vh height, which does not work in playground
+    due to the 50px header and 90px footer (50 + 90 = 140) */
+    #root>div {
```

Comment:
I think this should go into public/manager-styles.css within the Storybook project. Otherwise when we run locally (outside of the side) we won't see any affect.